purescript-prelude/Data_Show.go

```diff
@@ -2,8 +2,9 @@ package purescript_prelude
 
 import (
 	"fmt"
-	. "github.com/purescript-native/go-runtime"
 	"strings"
+
```

Comment:
`gofmt` 1.12.6 is automatically doing this for me, let me know if this is a problem.